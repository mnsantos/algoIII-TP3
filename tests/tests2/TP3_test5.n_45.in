45 331
1 10
2 10
3 10
5 10
1 2
1 8
1 7
2 11
1 3
5 12
5 7
4 17
5 14
2 18
2 3
8 9
3 7
1 4
4 11
3 13
9 10
5 13
9 13
7 8
7 12
1 16
1 17
1 11
4 12
4 7
2 15
1 18
7 13
5 8
1 9
43 45
42 43
41 45
41 44
41 42
40 41
39 45
38 43
38 41
38 39
37 43
37 41
37 40
37 38
36 45
36 41
36 37
35 43
35 39
35 38
34 41
34 37
33 44
33 41
33 39
33 36
33 35
32 41
32 40
32 37
32 34
31 45
31 44
31 39
31 38
31 37
31 34
31 33
30 38
30 36
30 35
30 31
29 43
29 39
29 37
29 36
29 35
29 33
28 37
28 36
28 31
28 30
27 42
27 36
27 32
27 31
26 45
26 44
26 43
26 39
26 33
26 31
26 30
26 27
25 38
25 37
25 36
25 33
25 32
24 43
24 40
24 31
24 30
24 29
23 36
23 32
23 27
22 44
22 42
22 41
22 39
22 37
22 36
22 35
22 32
22 23
21 42
21 38
21 37
21 35
21 34
21 32
21 29
21 26
21 25
21 23
20 45
20 39
20 36
20 34
20 33
20 32
20 30
20 26
20 25
20 24
20 23
19 41
19 40
19 39
19 37
19 36
19 30
19 28
19 27
19 22
18 44
18 43
18 31
18 26
18 20
17 45
17 44
17 42
17 40
17 37
17 34
17 27
17 26
17 24
17 22
17 21
16 44
16 42
16 40
16 37
16 35
16 33
6 22
5 31
5 30
5 29
5 21
5 20
5 17
4 44
4 34
3 44
3 43
3 37
3 34
3 32
3 31
3 29
3 28
3 26
3 24
3 21
2 42
2 41
2 40
2 36
2 35
2 33
2 26
2 25
2 20
2 19
1 26
1 25
1 24
1 23
1 22
1 21
1 20
1 19
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
1 36
1 37
1 38
1 39
1 40
1 41
1 42
1 43
1 44
1 45
6 26
6 27
6 34
6 35
6 40
6 44
7 19
7 20
7 21
7 24
7 25
7 27
7 29
7 31
7 35
7 39
8 21
8 23
8 26
8 30
8 31
8 33
8 41
8 42
9 14
9 20
9 21
9 25
9 31
9 35
9 40
10 15
10 20
10 21
10 22
10 24
10 28
10 29
10 31
10 32
10 41
10 42
10 43
10 44
11 13
11 19
11 20
11 21
11 26
11 31
11 34
11 42
12 15
12 17
12 21
12 24
12 27
12 30
12 31
12 32
12 35
12 39
12 41
12 43
13 24
13 28
13 32
13 33
13 36
13 39
13 41
13 44
14 19
14 20
14 21
14 23
14 27
14 33
14 35
14 41
15 20
15 36
15 38
15 41
15 43
16 21
16 22
16 24
16 29
16 31
1 15
3 14
6 13
1 13
2 12
1 6
1 12
1 14
3 5
1 5
3 16
0
