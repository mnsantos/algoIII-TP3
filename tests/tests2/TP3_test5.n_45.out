101 6 1 3 5 10 21 29
